build.ps1

```diff
@@ -0,0 +1,78 @@
+<#
+    .Synopsis
+        Convert the grammar file to json and/or run tests.
+
+    .DESCRIPTION
+        Build: Converts the .tmLanguage file to .json for use in VSCode or Atom.
+
+        Test: Uses Atom and atom-grammar-test to run tests against specs and sample files to guard against regression.
+
+    .EXAMPLE Build
+        PS C:\EditorSyntax> .\build.ps1
+        [Starting] Converting ../PowerShellSyntax.tmLanguage to json.
+        ... Reading source file.
+        ... Parsing content.
+        ... Building contents.
+        ... Creating directory: ./grammars
+        [Finished] File written to: ../grammars/powershell.tmLanguage.json
+
+    .EXAMPLE Test
+        PS C:\EditorSyntax> .\build.ps1 Test
+        Running specs...
+
+        .....
+
+        Finished in 0.281 seconds
+        2 tests, 5 assertions, 0 failures, 0 skipped
+#>
+[cmdletbinding()]
+param(
+    [parameter()]
+    [ValidateSet('Build','Test')]
+    [string]
+    $Task = 'Build'
```

Comment:
Do you think you could modify this build script to be more consistent to the other repo's build.ps1?

Take a look at https://github.com/PowerShell/Polaris/blob/master/build.ps1

```powershell
param(
    [Parameter()]
    [switch]
    $Bootstrap,  # prints out what needs to be installed

    [Parameter()]
    [switch]
    $Test,

    [Parameter()]
    [switch]
    $Package 
)
```

and then running it by itself will build.